python/ray/util/client/server/server.py

```diff
@@ -155,6 +155,7 @@ def ClusterInfo(self, request,
                 ctx.node_id = rtc.node_id.binary()
                 ctx.capture_client_tasks = \
                     rtc.should_capture_child_tasks_in_placement_group
+                ctx.runtime_env = pickle.dumps(rtc.runtime_env)
```

Comment:
Any reason to use pickle instead of JSON?

Reply:
I'm thinking that right now it's json, but later we probably want to use RuntimeEnvDict.

I'm ok with `json.dump` if you like.